plugin/output/s3/s3.go

```diff
@@ -23,6 +24,8 @@ const (
 	outPluginType      = "s3"
 	fileNameSeparator  = "_"
 	attemptIntervalMin = 1 * time.Second
+
+	bucketField = "bucket_name"
```

Comment:
what do you think about putting thin in config ?

Reply:
yes, it'll be useful

Reply:
Great idea, done.